### ISSUE TITLE
rollup throws a lot of warnings

### ISSUE THREAD

**Issue by rolandjitsu** [driftyco/ionic-app-scripts#444]
**Note: for support questions, please use one of these channels:**

https://forum.ionicframework.com/
http://ionicworldwide.herokuapp.com/

#### Short description of the problem:
Just wondering why is rollup throwing so many warnings that don't make sense:
```shell
> @infarm/operator@0.0.0 ionic:serve /Users/rolandjitsu/Projects/infarm/operator
> ionic-app-scripts serve

[14:49:53]  ionic-app-scripts 0.0.45 
[14:49:53]  watch started ... 
[14:49:53]  build dev started ... 
[14:49:53]  clean started ... 
[14:49:53]  clean finished in 6 ms 
[14:49:53]  copy started ... 
[14:49:53]  transpile started ... 
[14:49:57]  transpile finished in 4.10 s 
[14:49:57]  rollup started ... 
[14:49:58]  copy finished in 4.24 s 
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 'TextAst' 
            from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 
            'BoundTextAst' from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 'AttrAst' 
            from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 
            'BoundElementPropertyAst' from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 
            'BoundEventAst' from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 
            'ReferenceAst' from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 'VariableAst' 
            from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 'ElementAst' 
            from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 
            'EmbeddedTemplateAst' from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 
            'BoundDirectivePropertyAst' from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 
            'DirectiveAst' from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 'ProviderAst' 
            from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 
            'ProviderAstType' from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 
            'NgContentAst' from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 
            'PropertyBindingType' from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:00]  rollup: Conflicting namespaces: 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/index.js re-exports 
            'templateVisitAll' from both 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js
            (will be ignored) and 
            /Users/rolandjitsu/Projects/infarm/operator/node_modules/@angular/compiler/src/template_parser/template_ast.js.
[14:50:10]  rollup finished in 12.91 s 
```
It basically tells that two files (which is actually the same file - the strange part) export the same namespace and then it gets imported in index.

This might not be the place to create this issue, but I want confirmation that it's not some config in `config/rollup.config.js` that causes this.

#### What behavior are you expecting?
It should not throw any of these warnings.

**Steps to reproduce:**
1.
2.
3.

```
insert any relevant code between the above and below backticks
```

**Which @ionic/app-scripts version are you using?**

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)
Note the the production build is actually not throwing any of these.

**Comment by danbucholtz** [comment]
This is an issue Angular introduced. Should be fixed in the next Angular release. I'm going to close this for now as there's nothing we can do. We could swallow the warning but we don't think that's a good idea at this time.

Thanks,
Dan

**Comment by rolandjitsu** [comment]
I see, maybe it is already fixed as Ionic `2.0.0-rc.3` is still fixed on Angular `2.1.1` and Angular has a few more releases since then.
